ui/components/Navigator.js

```diff
@@ -319,10 +315,10 @@ const categories = [
     ],
   },
   {
-    id: "Management",
-    icon: <FontAwesomeIcon icon={faTerminal} transform="shrink-4" style={drawerIconsStyle} />,
+    id: "Lifecycle",
+    icon: <img src="/static/img/drawerIcons/lifecycle_mgmt.svg" transform="shrink-4"  style={drawerIconsStyle} />,
     href: "/management",
-    title: "Management",
+    title: "Lifecycle",
```

Comment:
Just changed the title from `Management` to `Lifecycle`.
Will have to update all the endpoints name from `management` to `lifecycle` or something similar. If the name is approved.

Reply:
Need approval on this too:
[open image](https://user-images.githubusercontent.com/48255244/126503618-c5624703-1fe5-41ff-9032-cf5139b1babe.png)

Reply:
The screenshot here is outdated, right? We still have the separator between MeshMap (an extension) and the other menu items, right?

Reply:
No I will put that in. Actually, I was signing with none provider and didn't got meshmap there. Hopefully it will solved now.